src/containers/ImportResources/ImportResources.js

```diff
@@ -226,6 +240,24 @@ export class ImportResources extends Component {
 
     return (
       <div className="outer">
+        {this.state.submitError && (
+          <InlineNotification
+            kind="error"
+            title={intl.formatMessage({
+              id: 'dashboard.error.title',
+              defaultMessage: 'Error:'
+            })}
+            subtitle={getErrorMessage(this.state.submitError)}
+            iconDescription={intl.formatMessage({
+              id: 'dashboard.notification.clear',
+              defaultMessage: 'Clear Notification'
+            })}
+            className="notificationComponent"
```

Comment:
I see this class name on many of the notifications but it's not actually used anywhere. Let's remove it from these new notifications, and we can follow up with the others later.

Reply:
ok, removed those now :)